<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialViewer</class>
 <widget class="QWidget" name="MaterialViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_Title">
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Material 0</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="minimumSize">
          <size>
           <width>325</width>
           <height>398</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>325</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Header</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Name Ptr:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_NamePtr"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Ref Ptr:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Ref Name:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Unknowns:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>State A:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>State B:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Texture Count:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_Name"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_RefPtr"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_RefName"/>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit_Unknowns"/>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEdit_StateA"/>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="lineEdit_StateB"/>
          </item>
          <item row="7" column="1">
           <widget class="QSpinBox" name="spinBox_TextureCount"/>
          </item>
          <item row="8" column="1">
           <widget class="QSpinBox" name="spinBox_ConstCount"/>
          </item>
          <item row="9" column="1">
           <widget class="QLineEdit" name="lineEdit_TechSetPtr"/>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Constant Count:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Tech Set Ptr:</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Texture Ptr:</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Constant Ptr:</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QLineEdit" name="lineEdit_TexturePtr"/>
          </item>
          <item row="11" column="1">
           <widget class="QLineEdit" name="lineEdit_ConstantPtr"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>400</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Data</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6"/>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>143</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
